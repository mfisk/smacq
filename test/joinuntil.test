#!/bin/sh

# This query prints out objects where b.srcip=192.168.0.200 and a.srcip is the a.srcip most recently seen before the packet from 192.168.0.200

${BINDIR}smacqq -g -m <<EOF 
print -v a.srcip b.srcip a.ipid b.ipid from pcapfile($TEST/0325@1112-snort.pcap) a until (new.a), b until (new.b) where (a.srcip=192.168.0.54 and b.srcip=192.168.0.200)
EOF
