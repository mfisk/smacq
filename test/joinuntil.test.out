a.srcip = 192.168.0.54	b.srcip = 192.168.0.200	a.ipid = 423	b.ipid = 41189
